doc/contribute/doc-guidelines.rst

```diff
@@ -0,0 +1,308 @@
+.. _doc_guidelines:
+
+Documentation Guidelines for the Zephyr Project
+###############################################
+
+Zephyr Project content is written using the `reStructuredText`_ markup
+language (.rst file extension) with Sphinx extensions, and processed
+using sphinx to create a formatted standalone website.  Developers can
+view this content either in its raw form as .rst markup files, or (with
+sphinx installed) they can run the makefile (or make.bat on windows) to
+generate the HTML content and view it with a web browser directly from
+their workstation's drive.  This same .rst content is also fed into the
+`Zephyr documentation`_ website (with a different theme applied).
+
+You can read details about `reStructuredText`_
+and about `Sphinx extensions`_ from their respective websites.
+
+.. _Sphinx extensions: http://www.sphinx-doc.org/en/stable/contents.html
+.. _reStructuredText: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html
+.. _Sphinx Inline Markup:  http://sphinx-doc.org/markup/inline.html#inline-markup
+.. _Zephyr documentation:  http://docs.zephyrproject.org
+
+This document provides a quick reference for commonly used ReST and
+sphinx-defined directives and roles used to create the documentation
+you're reading.
+
+Headings
+********
+
+For consistency, please use just an underline indicator for headings.
```

Comment:
From my reading of http://docutils.sourceforge.net/docs/user/rst/quickref.html#section-structure, this is the only option, so I don't understand the "for consistency" part. What am I missing?

Reply:
Titles can be written with a matching overline and underline (http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections) but we've standardized on only using the underline.

Reply:
Gotcha, thanks for explaining